clients/spark/core/src/main/scala/io/treeverse/gc/CommittedAddressLister.scala

```diff
@@ -25,8 +25,7 @@ class NaiveCommittedAddressLister extends CommittedAddressLister {
       .filter(
         (col("address_type") === AddressType.RELATIVE.name) ||
           // Backwards compatability with entries prior to address_type
-          (col("address_type") === AddressType.BY_PREFIX_DEPRECATED.name &&
-            !col("address_type").contains("://"))
+          col("address").startsWith(normalizedStorageNamespace)
```

Comment:
We should modify the comment above.

Reply:
True - I'll need to add code that will trim the storage prefix so the rest of the code will work the same with this physical address.